helm/domain-cc/templates/deployment.yaml

```diff
@@ -29,6 +29,8 @@ spec:
             {{- include "vro.commonEnvVars" . | nindent 12 }}
             {{- include "vro.mqClient.envVars" . | nindent 12 }}
             {{- include "vro.redisClient.envVars" . | nindent 12 }}
+            {{- include "domainCc.alembic.envVars" . | nindent 12 }}
+            {{- include "vro.postgresUrl" . | nindent 12 }}
```

Comment:
FYSA @tyler-spangler6 this allows us to make use of `DB_URL` from env vars (e.g. `os.environ.get()` in python)
See linked file [here](https://github.com/department-of-veterans-affairs/abd-vro/blob/032c4eca676bc22121573cba7033749c815b0c08/helm/_shared/named_templates/_postgres.tpl#L5).